src/main/java/p3/sramsbackend/metrics/Co2Metric.java

```diff
@@ -0,0 +1,21 @@
+package p3.sramsbackend.metrics;
+
+public class Co2Metric extends Metric {
+
+    final String metricQuery = "_room_co2_level";
```

Comment:
Husk @Override annotation
```suggestion
    @Override
    final String metricQuery = "_room_co2_level";
```

Reply:
Fikset